api/comments/views.py

```diff
@@ -119,6 +120,7 @@ class CommentDetail(JSONAPIBaseView, generics.RetrieveUpdateDestroyAPIView, Comm
                            "id":   {comment_id}, # required
                            "attributes": {
                              "content":       {content},        # mandatory
+                             "new_mentions":  {new_mentions},   # mandatory
```

Comment:
It's goofy that we say "required" just above, but "mandatory" here. I guess that's out of scope for this PR though.